### PR TITLE
Feature/edit tasks

### PR DESCRIPTION
**Pull request by maosan132**
# Description

Hello pals, I am re-submitting this pull request so everybody can give support and suggestions for making this project better, so it can be approved. Any suggestions will be fully appreciated. 

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -53,33 +42,56 @@
...
+          <i class="fas fa-minus-circle text-danger" role="button"></i>
+        </h3>
+      </div>
+      <hr class="m-0" />
+      <div class="d-flex justify-content-between">
+        <span>Need to make my head a new style for the party</span>
+        <h4 style="margin: 0px;"><i class="fa fa-battery-half"></i></h4>
+      </div>
     </div>
+  </template> -->
```
Comment (MuriloRoque) [inline dist/index.html]: - [ ] There are still some commented pieces of code.

File: src/helper.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+};
+
+const setAttributes = (el, attrs) => {
+  Object.entries(attrs).forEach(([key, value]) => {
+    el.setAttribute(key, value);
+  });
+  // for (const key in attrs) {
+  //   el.setAttribute(key, attrs[key]);
+  // }
+};
```
Comment (MuriloRoque) [inline src/helper.js]: - [ ] There are still some commented pieces of code.

File: src/project-model.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+// Project.prototype.edit = function () {
+// };
+
+
+// class Project {
+//   constructor(name, taskList = []) {
+//     this.name = name;
+//     this.taskList = taskList;|
+//   }
+// }
```
Comment (MuriloRoque) [inline src/project-model.js]: - [ ] There are still some commented pieces of code.

File: src/task-model.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+// /*eslint-disable */
+//     /* eslint-enable */
+// this.id = id;
+// this.name = name;
+// this.description = description;
+// this.date = date;
+// this.priority = priority;
+// this.status = status;
+//   }
+// }
```
Comment (MuriloRoque) [inline src/task-model.js]: - [ ] There are still some commented pieces of code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @maosan132 :wave: 

### Good job making some of the requested changes from the previous TSE. I really like your app, it's beautiful and functional! There are just some more things we need to fix before moving on:

- [ ] Please don't open another PR, you can continue using this one for the next review.
- [ ] Your live link is still redirecting to a "Not Found" page, please change it.
- [ ] Check my comments in your code, there are some commented pieces of code.
- [ ] Please fix your ESLint errors.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by maosan132** [comment]
Hi  MuriloRoque, thanks for your help. I have fixed everything.

**Review by bellom — APPROVED** [review]
## Hi @maosan132  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by maosan132** [comment]
Thanks so much

**Pull request closed without merging**